recipes/conversation_view/feed_item_row/feed_item_row_constants.js

```diff
@@ -0,0 +1,5 @@
+export const DEFAULT_CONSTANTS = null;
+
+export default {
+  DEFAULT_CONSTANTS,
+};
```

Comment:
Is this necessary somehow?, maybe we could just delet this file if we are not using it (I didnt search in the code if we are using it but Im guessing this is from the boilerplate)

Reply:
yeah it's just from the boilerplate generator, I don't really mind if it's there since it makes it easy to add constants when you need them.